advisor/src/test/kotlin/advisors/VulnerableCodeTest.kt

```diff
@@ -154,13 +152,14 @@ class VulnerableCodeTest : WordSpec({
                 val expStrutsVulnerabilities = listOf(
                     Vulnerability(
                         id = "CVE-2009-1382",
-                        url = URI("http://localhost:8000/api/vulnerabilities/v2/"),
-                        severity = 11.0f
+                        createReferences(
+                            url = URI("http://localhost:8000/api/vulnerabilities/v2/"),
+                            severity = 11.0f
+                        )
```

Comment:
Nit: This should still fit into a single line, like the other `createReferences` calls.